kernels/volk/volk_4ic_deinterleave_8i_x2.h

```diff
@@ -0,0 +1,109 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2020 Derek Kozel
+ *
+ */
+
+/*!
+ * \page volk_4ic_deinterleave_8i_x2
+ *
+ * \b Overview
+ *
+ * Deinterleaves the complex 4-bit integer vector into I & Q vector data
+ * and converts them to 8-bit integers.
+ *
+ * <b> Dispatcher Prototype</b>
+ * \code
+ * void volk_4ic_deinterleave_8i_x2(int8_t* iBuffer, int8t_t* qBuffer, const lv_4sc_t complexVector, unsigned int num_points) \endcode
```

Comment:
I assume the input vector is passed by pointer as well.

Reply:
Signature updated to the correct `int8_t *`